google-datacatalog-kafka-connector/tests/google/datacatalog_connectors/kafka/scrape/metadata_values_converter_test.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from google.datacatalog_connectors.kafka.scrape.metadata_values_converter \
+    import MetadataValuesConverter
+
+
+class MetadataValuesConverterTest(unittest.TestCase):
+
+    def test_duration_is_converted_to_human_redable_format(self):
+        input_in_ms = 31764000000
+        expected_output = '1 y 2 d 15 h 20 min'
```

Comment:
Wonder if those spaces make it easier or harder to understand. `1 y 2 d 15 h 20 min` vs. `1y 2d 15h 20min`

Reply:
I think without spaces it's easier to read, and it's better to remove spaces